### ISSUE TITLE
Erroneous mutation calls in version 0.9.0: AGC(Serine) => AAC(Asparagine not valine)

### ISSUE THREAD

**Issue by davidjstudholme** [Mykrobe-tools/mykrobe#156]
This issue is found when using the version at https://www.mykrobe.com/#download, i.e. version 0.9.0. Maybe this is fixed in version 0.11.0, but that later version is not offered on the mykrobe.com and does not include the GUI for running on Mac and Windows and therefore most users will still be using 0.9.0.

When analysing ERR1679585, Mykrobe 0.9.0 reports S315V in _katG_. 

![image](https://user-images.githubusercontent.com/3057078/169856425-11a23da8-06b1-4dea-8316-7fd42faefff3.png)

However, when we examine the genotype (with the help of IGV to visualise a BAM file representing an alignment of ERR1679585 versus the H37Rv reference genome) we see that the change is:

5' - G C T - 3'    =>  5' - G T T - 3'   
3' - C G A - 3'    =>  3' - C A A - 3'

In other words, since _katG_ is on the reverse strand, we get AGC( Serine) => AAC(Asparagine not valine)

![image](https://user-images.githubusercontent.com/3057078/169856293-444f3f54-b783-46f0-89c3-06395ca3245f.png)

So, Mykrobe should be reporting "S315N" not "S315V".

**Comment by martinghunt** [comment]
This was fixed in v0.10.0. Could you try downloading the v0.10.0 GUI from the github releases page please?
https://github.com/Mykrobe-tools/mykrobe/releases/tag/v0.10.0

We'll need to look separately into why mykrobe.com download links need updating, because as you say, they are currently pointing at 0.9.0 instead of 0.10.0.
